docs/code/getting-started.md

```diff
@@ -675,4 +675,162 @@ Let’s recap what we learned in this section:
+* The `translations` is the domain that should be installed and provide translations for the description
+* The `plugin` is the name of the plugin POA will use to handle the account
+
+As you can see, the biggest amount of lines are contained in the `template` key. They represent everything POA will need to authenticate, the required keys can variate between each methods and mechanisms so you might want to take a look at the keys that are supported by them.
+Nevertheless, a common part of the `template` is that you always have to specify the `auth/method` and `auth/mechanisms` settings, then you have to put all the supported keys in the `auth/$method/$mechanism` group. If any key is missing, the backend will fail and return you an explicit error message in the terminal.
+
+### Into the Build system {#poa-create-provider-build}
+
+In order to install the .provider file, you have to add a line to your "CMakeLists.txt" file previously created.
+
+        # install our .provider file
+        install (FILES ${CMAKE_CURRENT_SOURCE_DIR}/data/google.provider DESTINATION ${DATADIR}/accounts/providers/)
+
+## Create a Service {#poa-create-service}
+
+As in the [Create a Provider section](#poa-create-provider), there are several default services shipped with elementary, and much more available in the AppCenter. If the service you want is already available, [jump to the next part](#poa-declare-application) as you *MUST* use it. Otherwise, here is how to create a new one.
+The .service file should be created in the "data" directory of your project, it is better to name the file as `%provider-%service.service` (here it’s `google-contacts.service` for example)
+
+For our example, the .service file looks as following:
+
+        <?xml version="1.0" encoding="UTF-8"?>
+        <service id="google-contacts">
+            <type>contacts</type>
+            <name>Contacts</name>
+            <description>Sync your contacts</description>
+            <icon>office-address-book</icon>
+            <provider>google</provider>
+            <translations>my-address-book</translations>
+            
+            <template>
+                <group name="auth">
+                    <setting name="method">oauth</setting>
+                    <setting name="mechanism">user_agent</setting>
+                    <group name="oauth">
+                        <group name="user_agent">
+                            <setting type="boolean" name="ForceClientAuthViaRequestBody">true</setting>
+                            <setting name="AuthHost">accounts.google.com</setting>
+                            <setting name="TokenHost">accounts.google.com</setting>
+                            <setting name="AuthPath">/o/oauth2/auth</setting>
+                            <setting name="TokenPath">/o/oauth2/token</setting>
+                            <setting type="int" name="UiPolicy">0</setting>
+                            <setting name="RedirectUri">http://elementaryos.org</setting>
```

Comment:
Here the RedirectUri is listed as "elementaryos.org," yet above it is "elementary.io." I imagine that we want to be consistent. Also, it's probably worth mentioning to a new developer that the URI you choose isn't all that important, but that it MUST match the redirect URI you provide to the actual service provider (Twitter, Facebook, etc.) when you sign up to make an application (unless I'm wrong about that part).